unit-tests/native/src/test/scala/scala/scalanative/posix/sys/UdpSocketTest.scala

```diff
@@ -147,7 +147,7 @@ class UdpSocketTest {
         val maxInData = 2 * outData.length
         val inData: Ptr[Byte] = alloc[Byte](maxInData)
 
-        // Try to prevent spourious race conditions
+        // Try to prevent spourious race conditions.
```

Comment:
I think this should be `spurious`.

Reply:
Fixed now, thank you.  

I had added the terminal period as a way to force a restart.
I was concerned with the end of the line and did not even
see the typo, although I have seen it before whilst reading this code.

I will also correct the related `Udp6SocketTest,scala` that is my current
workpiece.

Correcting the typo gives me the fresh restart I wanted. A double Thank You.